src/components/shared/group_nav.cr

```diff
@@ -0,0 +1,10 @@
+module Shared::GroupNav
```

Comment:
Thoughts on using a Component here so it is clearer where the group nav is rendered?

Reply:
I've never made one! I'll give it a shot